### PR TITLE
Bug fix 2

### PR DESCRIPTION
**Pull request by dilin-MS**

### CODE REVIEW

File: resources/templates/boardlist.json [hunk 1]
```diff
@@ -38,14 +31,6 @@
                 "architecture": "esp32"
             },
             "projectHostType": "workspace"
-        },
-        {
-            "name": "Raspberry Pi",
-            "id": "raspberrypi",
-            "detailInfo": "Raspberry Pi",
-            "helpUrl": "https://www.raspberrypi.org/",
-            "exampleUrl": "https://raw.githubusercontent.com/VSChina/azureiotdevkit_tools/gallery/workbench-example-raspi.json",
```
Comment (lirenhe) [inline resources/templates/boardlist.json]: You could just remove the example Url here.
No need to remove Rpi as the supported device.

File: resources/templates/templates.json [hunk 2]
```diff
@@ -71,44 +71,9 @@
             "type": "AzureFunctions"
         },
         {
```
Comment (lirenhe) [inline resources/templates/templates.json]: I think there are other text you need to update.  like  'MXCHIP IoT DevKit'

File: src/AzureOperator.ts [hunk 3]
```diff
@@ -56,6 +57,12 @@ export class AzureOperator {
   async Deploy(
       context: vscode.ExtensionContext, channel: vscode.OutputChannel,
       telemetryContext: TelemetryContext) {
+    if (RemoteExtension.isRemote(context)) {
+      const message =
+          `You are in a container now. 'Azure IoT Device Workbench: Depoly to Azure...' command is currently not supported inside the container.`;
```
Comment (lirenhe) [inline src/AzureOperator.ts]: The project is currently open in container now. 'Azure IoT Device Workbench: Depoly to Azure...' is not supported inside the container.

File: src/exampleExplorer.ts [hunk 4]
```diff
@@ -186,6 +187,12 @@ export class ExampleExplorer {
   async selectBoard(
       context: vscode.ExtensionContext, channel: vscode.OutputChannel,
       telemetryContext: TelemetryContext) {
+    if (RemoteExtension.isRemote(context)) {
+      const message =
+          `You are in a container now. You can open examples in a new window(click 'File -> New Window').`;
```
Comment (lirenhe) [inline src/exampleExplorer.ts]: The project is open in Docker container now, Please open a new window and rerun this command.

File: src/extension.ts [hunk 5]
```diff
@@ -308,6 +289,12 @@ export async function activate(context: vscode.ExtensionContext) {
 
   const workbenchPath =
       vscode.commands.registerCommand('iotworkbench.workbench', async () => {
+        if (RemoteExtension.isRemote(context)) {
+          const message =
+              `You are in a container now. Setting project default path is currently not supported inside the container. You can do it in a new window(click 'File -> New Window').`;
```
Comment (lirenhe) [inline src/extension.ts]: Same above

File: src/projectInitializer.ts [hunk 6]
```diff
@@ -33,6 +32,13 @@ export class ProjectInitializer {
   async InitializeProject(
       context: vscode.ExtensionContext, channel: vscode.OutputChannel,
       telemetryContext: TelemetryContext) {
+    if (RemoteExtension.isRemote(context)) {
+      const message =
+          `You are in a container now. You can create a new project in a new window(click 'File -> New Window').`;
```
Comment (lirenhe) [inline src/projectInitializer.ts]: Same above

File: resources/templates/boardlist.json (hunk 1 above)
Comment (dilin-MS) [reply resources/templates/boardlist.json]: Fixed.

File: resources/templates/templates.json (hunk 2 above)
Comment (dilin-MS) [reply resources/templates/templates.json]: Fixed.

File: src/AzureOperator.ts (hunk 3 above)
Comment (dilin-MS) [reply src/AzureOperator.ts]: Fixed.

File: src/exampleExplorer.ts (hunk 4 above)
Comment (dilin-MS) [reply src/exampleExplorer.ts]: Fixed.

File: src/extension.ts (hunk 5 above)
Comment (dilin-MS) [reply src/extension.ts]: Fixed.

File: src/projectInitializer.ts (hunk 6 above)
Comment (dilin-MS) [reply src/projectInitializer.ts]: Fixed.

**Pull request merged**